src/css/components/_control.scss

```diff
@@ -36,3 +36,28 @@
   display: table-cell;
   vertical-align: middle;
 }
+
+// visually hide a control-bar component without changing the position
+// and without hiding it from screen readers. This also preserves
+// a transition animation.
+.video-js .vjs-control .vjs-visual-hide-vertical {
```

Comment:
We need to have two in order to correctly `transition` components like the volume bar that are hidden and show on the `contral-bar`. If we were to do the `width` and `heigth` at the same time the horizontal volume-bar would appear to move up and left as it transitions out which looked a little weird and was not the current behavior. A vertical volume bar would have a similar transition problem.

I looked at the other hiding techniques but the fact that they position the element in a different place means that we would lose the transition.